code/style.css

```diff
@@ -1,58 +1,140 @@
+* { 
+  box-sizing: border-box;
+ }
+ *:before,
+ *:after {
+  box-sizing: border-box;
+ }
+
+html,
+body {
+ height: 100%;
+ position: relative;
+}
+
 body {
   margin: 0;
-  font-family: "Courier New", Courier, monospace;
+  font-family: GabrielWeissFriendsFont;
+  background-image: url(./assets/background.jpg);
```

Comment:
It's far more safe approach to wrap path in parentheses. Without them, some browsers might not find the path
`background-image: url("./assets/background.jpg");`

Reply:
Definitely, I totally forget that!